### ISSUE TITLE
1.19 support?

### ISSUE THREAD

**Issue by VL4DST3R** [Fenixin/Minecraft-Region-Fixer#184]
**Is your feature request related to a problem? Please describe.**
Just wanted to make sure i don't nuke my worlds by running this on 1.19+ maps. In fact, id like to know if the tool is still being worked on at all or if it is abandoned? I made a ticket about some errors i was experiencing with some of my older maps (#151) and i was wondering if it ever came (or will) to anything. If not, at least i can stop monitoring this project XD. Thanks.

**Describe the solution you'd like**
Just some info for now, hopefully an update from the developer.

**Describe how the solution would be implemented**
Written, preferably in plain English.

**Describe alternatives you've considered**
Not using it and hoping for the best? lol

**Additional context**
n/a

**Comment by Fenixin** [comment]
Hello!

> **Is your feature request related to a problem? Please describe.** Just wanted to make sure i don't nuke my worlds by running this on 1.19+ maps. In fact, id like to know if the tool is still being worked on at all or if it is abandoned? I made a ticket about some errors i was experiencing with some of my older maps (#151) and i was wondering if it ever came (or will) to anything. If not, at least i can stop monitoring this project XD. Thanks.

I might sound a bit rude but this tool is being worked on when I feel like it and I have free time. I'm still maintaining it under those two conditions. So, you probably want to keep monitoring it, but take into acount that this is not my job, it's just a hobby that is not even related with my real job. And, although I still play videogames, I haven't played Minecraft in YEARS.

Talking about updates, I have an update ready that fixes #144 and #177, but I wanted to test it a little more. Maybe I push it now.

> **Describe the solution you'd like** Just some info for now, hopefully an update from the developer.
Here it is! An update!

> **Describe how the solution would be implemented** Written, preferably in plain English.
I can also provide it in spanish.

> **Describe alternatives you've considered** Not using it and hoping for the best? lol
You can do that!

> **Additional context** n/a

About 1.19: There are no world format changes in 1.19, so there is no problem at all. I tested regionfixer with a new 1.19 world and it doesn't seem to break anything. If you find anything sus, please, just tell me. I'm going to close this because I think this answer solves the issue. Don't hesitate to ask if you need anything else.

I hope that helps!

**Comment by VL4DST3R** [comment]
> > **Describe the solution you'd like** Just some info for now, hopefully an update from the developer.
> > Here it is! An update!
> 
> > **Describe how the solution would be implemented** Written, preferably in plain English.
> > I can also provide it in spanish.

> I hope that helps!

Great stuff XD Glad to hear you're still around. Will try to run it again after you push those two fixes. 

I'm guessing you haven't got the time to look over those errors i was encountering? It seems to be serious enough that not even the `--forceUpgrade` flag on the server can deal with all the maps i've been having trouble with (it locks the server up and never finishes converting).

That's why i was kinda putting all my faith in you figuring out whats going on with them, but i understand if you're busy. Hope you'll get some time eventually to help me out with this mess!
